src/api/InkAPI.cc

```diff
@@ -1518,17 +1518,18 @@ TSMimeHdrCopy(TSMBuffer dest_bufp, TSMLoc dest_obj, TSMBuffer src_bufp, TSMLoc s
 void
 TSMimeHdrPrint(TSMBuffer bufp, TSMLoc obj, TSIOBuffer iobufp)
 {
+  // The bufp argument is no longer used, but the assert is left in to verify
+  // the preconditon so that the API contract is not inadvertantly changed.
   sdk_assert(sdk_sanity_check_mbuffer(bufp) == TS_SUCCESS);
```

Comment:
Adding @zwoop to get his input. @JosiahWI's work here cleans up things and removes this unused `bufp` parameter. Well, it becomes unused after this cleanup. Do we want to change the `TSMimeHdrPrint` accordingly? Maybe add a new function without `bufp` with a deprecation on this old one with the intention have b removed in the 11 release?

Regardless, we do not need to hold up this PR. We can handle that as a separate PR.

Reply:
If we can get this in for 10.0.0 @cmcfarlen ?) I'd say fix the API now and break compatibility. This is really just for debugging as I understand things, so this is not going to break any thing critical. Obviously fix any plugins in the core accordingly.

That change would need to go to the mailing list as well, but we're certainly at liberty to break compatibility for 10.0.0. And again, this doesn't seem like an API worthy of making two versions of :).